jparest/src/main/java/uk/gov/homeoffice/digital/sas/jparest/controller/ResourceApiController.java

```diff
@@ -148,7 +150,7 @@ private T getById(UUID id, String include, UUID tenantId) {
 
     public ApiResponse<T> get(@RequestParam UUID tenantId, @PathVariable UUID id) {
         var result = getById(id, null, tenantId);
-        return new ApiResponse<>(Arrays.asList(result));
+        return new ApiResponse<>(List.of(result));
```

Comment:
Since we're working in this area, might be cleaner to add another consturctor to ApiResponse that takes a single result and adds it to it's item list. Since we commonly add only a single result object.

Reply:
Good point Daleel. Unfortunately I missed your comment by a minute and had already merged.
We should include this next time we're touching jparest